src/components/InterimFindings/index.js

```diff
@@ -88,14 +88,31 @@ export default function InterimFindings() {
             <span>100</span>
           </div>
         </div>
+        <div className="wrapRound">
```

Comment:
ui를 안봐서 모르지만 div보단 섹션으로 분리해도 좋을 것 같아 보입니다.
```suggestion
        <section className="wrapRound">
```

Reply:
한번 고민해보고 tag 변경해보겠습니다.